modules/tinymce/src/plugins/table/test/ts/browser/ui/TableClassListButtonsTest.ts

```diff
@@ -0,0 +1,126 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinySelections, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+import Editor from 'tinymce/core/api/Editor';
+import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
+import { TableModifiedEvent } from 'tinymce/plugins/table/api/Events';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.table.ui.TableClassListButtonsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    table_class_list: [
+      {
+        title: 'A',
+        value: 'a'
+      },
+      {
+        title: 'B',
+        value: 'b'
+      }
+    ],
+    toolbar: 'tableclass',
+    base_url: '/project/tinymce/js/tinymce',
+    setup: (editor: Editor) => {
+      editor.on('tablemodified', logEvent);
+    }
+  }, [ Plugin, Theme ], true);
+
+  let events: Array<EditorEvent<TableModifiedEvent>> = [];
+  const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
+    events.push(event);
+  };
+
+  afterEach(() => {
+    events = [];
+
+    try { // Check if the menu is still open. If it is, close it.
```

Comment:
Why is this in a try-catch loop? This has a massive code smell about it and it would appear to be done because individual tests aren't cleaning up.

- [ ] Ideally remove this and make sure menus are closed in the tests where they are opened